pyproject.toml

```diff
@@ -9,6 +9,7 @@ dependencies = [
     "pandas==2.2.2",
     "polars[pandas]==0.20.31",
     "scikit-learn==1.5.0",
+    "typing-extensions==4.12.2",
```

Comment:
`typing_extensions` is a package that is automatically available for python version < 3.10. This is not the way to solve the typing problem

Reply:
>3.10?

Reply:
nope versions < 3.10
After that it is available in the `typing package`

Reply:
@LauraBoenchenLB ey lösch doch dein kommentar nich einfach. Pff